src/controllers/article.js

```diff
@@ -0,0 +1,33 @@
+import { slug } from '../utils/article';
+import { Article } from '../models';
+
+/**
+ * @name AddArticles
+ * @description This is the method for inserting articles
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {int} Returns the inserted article after success
+ */
+export const AddArticles = async (req, res) => {
+  const { body } = req;
+  const article = await Article.create({ ...body, slug: slug(body.title) });
+  return res.status(201).json(article);
```

Comment:
@chikeozulumba what happens when a database failure occurs? Why are we not handling errors?